### ISSUE TITLE
Notification is not floating on screen

### ISSUE THREAD

**Issue by theunreal** [flauc/angular2-notifications#44]
If my browser is not scrolled to the bottom of the web page, I can't see the notifications. 

The noficiation is not working as "fixed" in my app, although it works fine in the provided demoe.

What can cause it to happen?

**Comment by flauc** [comment]
Hi @theunreal 

There must be some conflict with the css. I've added `encapsulation: ViewEncapsulation.None` because of an user request. So if you have a class with the same name it applies to the notification as well.

**Comment by theunreal** [comment]
@flauc I didn't get from your answer what could have gone wrong and how encapsulation has to do with it?

**Comment by flauc** [comment]
You might have the same class i use defined in some other css file.

**Comment by flauc** [comment]
Inspect the simple notification wrapper and check if the following are the only styles attached to the element:

```css
        .simple-notification-wrapper {
            position: fixed;
            bottom: 20px;
            right: 20px;
            width: 300px;
            z-index: 1000;
        }
```

**Comment by flauc** [comment]
Hi, did you manage to figure out what the problem was?

**Comment by theunreal** [comment]
Nope, I don't have any class named `.simple-notification-wrapper` or anythign similar in my app, I have no idea why it happens \:

**Comment by flauc** [comment]
Can you try to recreate the error in a plunker?

**Comment by theunreal** [comment]
hey @flauc , this error happens when using the notifications in Angular2-material Sidenav content.
I tried to plunker it but failed with importing both of the packages in a plunker and got messy.

The notifications floating is not working when using the angular 2 material sidenav and the notifications.
Works only if ```
<simple-notifications [options]="options">
</simple-notifications>
```

is placed outside my sidenav, but then I can't use it in my components..

**Comment by mtrdp642** [comment]
I was able to get a plunker working that demostrates this issue (warning: it's not pretty). Click the "Toggle Start Side Drawer" button to open the sidenav and "Create Notification" to create a notification that is only visible if you scroll to the bottom of the sidenav.

[plunker](https://plnkr.co/edit/ZjYtptumVZiUsjFZ5217)

@theunreal, this perhaps is an issue on the angular2-material sidenav control and how things are rendered within it. It may be worth creating an issue on their page as well.

**Comment by flauc** [comment]
Thank you for the effort 👍 

Let's see if i can dig something up

**Comment by flauc** [comment]
Okay so as far as i can figure this has nothing to do with the notifications library. It has something to do with the styling. 

If you set the position in the notifications options to be `position: ['right', 'top']` you will see them at the top of the screen. Here is the example: 

https://plnkr.co/edit/CSmsoWAlWdouZKe5PrmM

**Comment by flauc** [comment]
I've managed to solve the issue. Here is the new plunker with the fixed position working properly: 

https://plnkr.co/edit/G1sNSovEChY3cuI02QuT?p=preview

What i did was. I moved the `<simple-notifications [options]="options"></simple-notifications>` component from the `<md-sidenav-layout></md-sidenav-layout>` there is something in that component that is causing the error. However it's `position: fixed` so there is no reason why you would place it in the layout element.
